lib/k8s/resource_client.rb

```diff
@@ -186,10 +195,12 @@ def list?
 
     # @param list [K8s::API::MetaV1::List]
     # @return [Array<Object>] array of instances of resource_class
-    def process_list(list)
+    def process_list(list, namespace: @namespace)
```

Comment:
this method does not seem to use the `namespace:` keyword.

Reply:
Bug.